docs/standard/commandline/model-binding.md

```diff
@@ -157,12 +157,30 @@ With `FileInfo` and `DirectoryInfo` the pattern matching code is not required:
 
 :::code language="csharp" source="snippets/model-binding/csharp/Program.cs" id="filesysteminfo" :::
 
-### Anything with a string constructor
+### Other supported types
 
-The file system types aren't special cases. Any type having a constructor that takes a single string parameter can be bound in this way. For example, code that would work with `FileInfo` works with a <xref:System.Uri> instead.
+Many types that have a constructor that takes a single string parameter can be bound in this way. For example, code that would work with `FileInfo` works with a <xref:System.Uri> instead.
 
 :::code language="csharp" source="snippets/model-binding/csharp/Program.cs" id="uri" :::
 
+Besides the file system types and `Uri`, the following types are supported:
```

Comment:
I expect this list will continue to grow until we can find a generalized approach that's also supportable via NativeAOT compilation.

Reply:
I suspect we'll end up at the static interfaces feature that the C# team is working on as a long-term, reflection-free solution. But that's just my guess.